### PR TITLE
Inventory and playerlist text api

### PR DESCRIPTION
**Pull request by Creeplays**

### CODE REVIEW

**Comment by Creeplays** [comment]
Latest commit to add modpe-squid support (https://github.com/Creeplays/modpe-squid/commit/609c37409c861e5f3b06613d5e33b4dfdadefe88)

**Comment by rom1504** [comment]
Err why did you put back we.js and in particular why in that PR ? the other PR was better

**Comment by Creeplays** [comment]
No, there modpe support is server plugin and should work better.

**Comment by Creeplays** [comment]
Will be merged?

**Comment by rom1504** [comment]
I really think we.js shouldn't be in this repo.

**Comment by Creeplays** [comment]
Removed

File: lib/serverPlugins/modpe.js [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    console.log("[MODPE-NOINJECT] " + msg);
+}
+
+function modpeApi() {
+    function euclideanMod(numerator, denominator) {
+        var result = numerator % denominator;
+        return result < 0 ? result + denominator : result;
+    }
+
+    function Vec3(x, y, z) {
```
Comment (rom1504) [inline lib/serverPlugins/modpe.js]: Why are you redefining Vec3 here instead of using the vec3 you required at line 1 ?

**Comment by Creeplays** [comment]
Thats for modpe scripts, I dont want to have node-modules in modpePlugins dir.

**Comment by Creeplays** [comment]
All contents from function modpeApi() are used in runtime by modpe scripts, modpe doesn't have to global scope.

**Comment by rom1504** [comment]
copy/pasting the code of dependencies cannot be the right solution.

**Comment by Creeplays** [comment]
But that is best solution for this.

**Comment by rom1504** [comment]
No, it's the best solution you found but even reading node_modules/node_vec3/index.js and evaluating the resulting string is better

**Comment by Creeplays** [comment]
Emm...

File: examples/basic.js [hunk 2]
```diff
@@ -1,11 +0,0 @@
-var mcServer=require("..");
```
Comment (rom1504) [inline examples/basic.js]: hmm why did you delete that file ?

**Comment by Creeplays** [comment]
I am unecessary.

**Comment by Creeplays** [comment]
Accidentally*

**Comment by Creeplays** [comment]
Yes, i am dont know English...

**Comment by rom1504** [comment]
Can you rebase this ? (`git fetch` then `git rebase master` then solve the conflicts then `git push --force`)
Thanks

**Comment by rom1504** [comment]
Alright so, I'm testing it.
The inventory change doesn't seem to work (connect 2 accounts, hold an item with one, the other doesn't see it).

**Comment by Creeplays** [comment]
Seems like inventory work only in creative...

**Comment by rom1504** [comment]
I fixed that, I'll push the changes soon ;)

**Comment by rom1504** [comment]
@Creeplays so I got it all working, it's nice. Where is we.js coming from ? Is it stored somewhere ?

**Comment by rom1504** [comment]
Merged ! Thanks @Creeplays 

location of modpe plugins handled there https://github.com/mhsjlw/flying-squid/issues/65
